openflow15/match.go

```diff
@@ -0,0 +1,2173 @@
+package openflow15
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"log"
+	"net"
+
+	"antrea.io/libOpenflow/util"
+)
+
+// ofp_match 1.5
+type Match struct {
+	Type   uint16
+	Length uint16
+	Fields []MatchField
+}
+
+// One match field TLV
+type MatchField struct {
+	Class          uint16
+	Field          uint8
+	HasMask        bool
+	Length         uint8
+	ExperimenterID uint32
+	Value          util.Message
+	Mask           util.Message
+}
+
+// oxm_id - OXM TLV Header
+type OxmId struct {
```

Comment:
Is this expected to use in flow action part to represent a different field? In OpenFlow13, I use MatchField for this purpose, please add some functions to help users to create the target OxmId with a name string (like the existing function FindFieldHeaderByName ) if you want it for the same purpose. It is because we can't expect all dev know every field's class, field, length, experimenterID...